nd4j/nd4j-backends/nd4j-api-parent/nd4j-api/src/main/java/org/nd4j/linalg/api/ops/impl/reduce/Mmul.java

```diff
@@ -93,6 +93,22 @@ public Mmul(INDArray x,
         }
     }
 
+    public Mmul(INDArray x, INDArray y, boolean transposeX, boolean transposeY,  boolean transposeZ) {
+        addInputArgument(x, y);
+        addBArgument(transposeX, transposeY, transposeZ);
+    }
+
+    public Mmul(INDArray x, INDArray y) {
+        this(x,y,null,null);
+    }
+
+    public Mmul(SameDiff sameDiff, SDVariable x, SDVariable y, boolean transposeX, boolean transposeY,
+                boolean transposeZ) {
+        super(null,sameDiff,new SDVariable[]{x,y});
+        addIArgument(ArrayUtil.fromBoolean(transposeX),
+                     ArrayUtil.fromBoolean(transposeY),
+                     ArrayUtil.fromBoolean(transposeZ));
```

Comment:
INDArray constructor uses BArgs, SDVariable constructor uses IArgs? Should be same... check this

Reply:
Fixed.